scripts/CleanCluster.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env sh
+
```

Comment:
Can you please add a brief comment indicating the purpose of the code in this file? Also, should the name reflect that it's a script about cleaning namespaces on the cluster? `CleanCluster` kinda sounds like it's going to clean the cluster itself.

Reply:
Comment added, also updated the name of the script